src/clr-angular/forms/combobox/combobox.html

```diff
@@ -1,32 +1,70 @@
+    <!--<div id="foo" style="display: none;">Type and press the down arrow to browse available matches</div>-->
+    <ng-container *ngIf="multiSelect">
+      <!--used to describe the current selection when input receives focus-->
+      <span [id]="ariaDescribedBySelection" class="clr-aria-only" *ngIf="optionSelectionService.selectionModel.model">
+          {{commonStrings.keys.selection + ' ' + optionSelectionService.selectionModel.toString(displayField)}}
+        </span>
 
-<!-- Content projected by the consumer in ng-template. -->
-<ng-template #projectedOptions>
-    <ng-content></ng-content>
-</ng-template>
+      <span *ngIf="optionSelectionService.selectionModel.model">
+        <span *ngFor="let item of optionSelectionService.selectionModel.model; let i = index"
+              class="label label-combobox-pill">
+          <span tabindex="0" class="clr-combobox-pill-content">
+            <ng-container *ngIf="optionSelected" [ngTemplateOutlet]="optionSelected.template"
+                          [ngTemplateOutletContext]="{$implicit: optionSelectionService.selectionModel.model[i]}">
+            </ng-container>
+          </span>
+          <button type="button" class="clr-combobox-remove-btn" [disabled]="control?.disabled? true: null"
+                  [attr.aria-label]="commonStrings.keys.delete + ' ' + optionSelectionService.selectionModel.toString(displayField, i)"
+                  (click)="unselect(item)"><clr-icon shape="close" size="12"></clr-icon></button>
+        </span>
+      </span>
+      <ng-container [ngTemplateOutlet]="textbox"></ng-container>
+
+    </ng-container>
+
+    <ng-container *ngIf="!multiSelect">
+      <ng-container *ngIf="showCurrentSingleSelection && optionSelectionService.selectionModel">
 
-<!-- Create the options menu ourself if the consumer doesnt provide the menu. -->
-<clr-options *ngIf="!options">
-    <ng-container [ngTemplateOutlet]="projectedOptions"></ng-container>
-</clr-options>
+        <!--used to describe the current selection when input receives focus-->
+        <span [id]="ariaDescribedBySelection" class="clr-aria-only" *ngIf="optionSelectionService.selectionModel.model">
+          {{commonStrings.keys.selection + ' ' + optionSelectionService.selectionModel.toString(displayField)}}
+        </span>
 
-<!-- If consumer provides the menu, render it -->
-<ng-container *ngIf="options" [ngTemplateOutlet]="projectedOptions"></ng-container>
+        <span *ngIf="optionSelected" [ngTemplateOutlet]="optionSelected.template"
+              [ngTemplateOutletContext]="{$implicit: optionSelectionService.selectionModel.model}">
+        </span>
+        <span *ngIf="!optionSelected" class="clr-selection">{{optionSelectionService.selectionModel.model}}</span>
+      </ng-container>
+      <ng-container [ngTemplateOutlet]="textbox"></ng-container>
+    </ng-container>
+
+    <button clrPopoverOpenCloseButton #trigger type="button" class="clr-combobox-trigger" tabindex="-1">
```

Comment:
Is there a reason users cannot tab to this button?

Reply:
You can't focus it in the W3C examples too:
https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.0pattern/combobox-autocomplete-list.html